udata/forms/fields.py

```diff
@@ -80,6 +80,19 @@ class IntegerField(FieldHelper, fields.IntegerField):
     pass
 
 
+class RolesField(FieldHelper, fields.StringField):
+    def process_formdata(self, valuelist):
+        self.data = []
+        if valuelist:
```

Comment:
Does the default `valuelist` return `None`? Otherwise iterating over an empty list is OK.